compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -561,7 +561,7 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager, bool packPeel) {
   passManager.addPass(xilinx::air::createAIRLoweringPass());
   {
     xilinx::air::AffineLoopOptPassOptions options;
-    const std::vector<unsigned> tile_sizes = {4, 4};
+    const std::vector<unsigned> tile_sizes = {2, 2};
```

Comment:
I think while {2,2} works with both pad-pack and pack-peel pipelines through AIR, {4,4} can also work if pad-pack; pack-peel requires {2,2}. 

{4,4} is faster due to having less NPU instructions. So it might be beneficial for performance if we do a condition here between pad-pack and pack-peel, so that pad-pack can still use {4,4} for now.